manifests/pipecd/grafana-dashboards/piped/go.json

```diff
@@ -290,7 +290,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum by (piped) (go_memstats_heap_alloc_bytes{job=\"pipecd-ops\", pipecd_component=\"piped\"} / (1024 * 1024))",
+          "expr": "sum by (piped) (go_memstats_heap_alloc_bytes{job=\"pipecd-ops\", pipecd_component=\"piped\", piped=~\"[[piped]]\"} / (1024 * 1024))",
```

Comment:
Enclosing with `[[ ]]` is deprecated syntax. Let's use "$var" according to other dashboards.
https://grafana.com/docs/grafana/latest/variables/syntax/

Reply:
Oh I see, just get that from here [grafana/docs](https://grafana.com/docs/grafana/latest/datasources/prometheus/#using-variables-in-queries) since I need the ALL option 👀  Will try out your suggestion 👍

Reply:
addressed by https://github.com/pipe-cd/pipe/pull/2326/commits/8e72f404e3550df4861921c24b8f352002eb595b 🙏 y